docs/example_ScalarField_Geotiff.html

```diff
@@ -11,19 +11,19 @@
 </head>
 
 <body>
-    <h1 class="title mapTitle">ScalarField GeoTIFF</h1>
```

Comment:
Please, try to avoid these little format changes, so I can focus more on real important changes in your code (I use VSCode + Prettier)

Reply:
This change was unintentional, sorry.